local_planner/src/nodes/local_planner_node.cpp

```diff
@@ -122,7 +120,7 @@ LocalPlannerNode::~LocalPlannerNode() { delete server_; }
 
 void LocalPlannerNode::readParams() {
   // Parameter from launch file
-  auto goal = local_planner_->getGoal();
+  auto goal = toPoint(local_planner_->getGoal());
```

Comment:
Rather do the toPoint down below when it is actually going into the goal_msg_, and up here use an Eigen type

Reply:
goal needs to be a geometry_msgs type because of the launch params assignments in the lines below.